bindata/network/ovn-kubernetes/ipsec.yaml

```diff
@@ -207,6 +207,14 @@ spec:
           # Start the pluto IKE daemon
           /usr/libexec/ipsec/pluto --leak-detective --config /etc/ipsec.conf --logfile /var/log/openvswitch/libreswan.log
 
+          # IBMCloud does not forward ESP (IP proto 50)
+          # Instead, force IBMCloud IPsec to always use NAT-T
+          platformType=$(kubectl get infrastructure cluster  -o jsonpath='{.spec.platformSpec.type}')
+          if [ "${platformType}" == "IBMCloud" ]; then
+              if ! grep -q encapsulation=yes /usr/share/openvswitch/scripts/ovs-monitor-ipsec ; then
```

Comment:
This is obviously *not* the solution. We'd ideally have a CLI parameter here: https://github.com/openvswitch/ovs/blob/master/ipsec/ovs-monitor-ipsec.in#L167